src/App.js

```diff
@@ -1,19 +1,45 @@
 import Clothing from './components/Clothing'
 import styled from 'styled-components/macro'
-import data from './mockup-data'
+import mockupData from './mockup-data'
 
 function App() {
+  let data
+
+  if (localStorage.getItem('localClothing')) {
+    data = JSON.parse(localStorage.getItem('localClothing'))
+  } else {
+    localStorage.setItem('localClothing', JSON.stringify(mockupData))
+    data = JSON.parse(localStorage.getItem('localClothing'))
+  }
+
+  const handleBookmark = (bookmarkActive) => {
+    const index = data.findIndex((card) => card.id === bookmarkActive.id)
+
+    localStorage.setItem(
+      'localClothing',
+      JSON.stringify([
+        ...data.slice(0, index),
+        bookmarkActive,
+        ...data.slice(index + 1),
+      ]),
+    )
+  }
```

Comment:
I added a note in our course channel about how to implement `localStorage` with a bit better readability. Maybe have a look and change your implementation - if you want 🙂

Reply:
It works 🙂 Thanks for inspiration!